ppr-ui/src/components/mhrTransfers/TransferDetails.vue

```diff
@@ -148,9 +148,9 @@ export default defineComponent({
         customRules(maxLength(80), props.validateTransferDetails ? required('Enter consideration') : [])
     )
 
-    const updateCompensation = (declaredValue: number) => {
+    const updateCompensation = (declaredValue: any) => {
       // copy Declared Value into Consideration field - the initial time only
-      if (!localState.consideration && localState.declaredValue) {
+      if (!localState.consideration && localState.declaredValue && parseInt(declaredValue)) {
```

Comment:
If `parseInt` is required, wouldn't `declaredValue` be of type `string`?

Reply:
Updated!